src/parts/Sidebar/SidebarContent/GetGovernanceTokenUI/index.tsx

```diff
@@ -0,0 +1,155 @@
+
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import clsx from 'clsx';
+
+import TitleWithUnderline from 'components/TitleWithUnderline';
+import InterlayDefaultOutlinedButton from 'components/buttons/InterlayDefaultOutlinedButton';
+import InterlayModal, { InterlayModalInnerWrapper } from 'components/UI/InterlayModal';
+import InterlayLink from 'components/UI/InterlayLink';
+import { GOVERNANCE_TOKEN_SYMBOL } from 'config/relay-chains';
+import { ReactComponent as GateLogoIcon } from 'assets/img/exchanges/gate-logo.svg';
+// ray test touch <<
+import { ReactComponent as KrakenLogoIcon } from 'assets/img/exchanges/kraken-logo.svg'; // TODO: placeholder
+// ray test touch >>
```

Comment:
For now, I'm waiting for a better version. Once it's replaced with it, let me remove the comments.

Reply:
Done!